### PR TITLE
SEARCH-1138: update README for alfresco-search-deployment

### PR DESCRIPTION
**Pull request by paulbrodner**
Updating README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+## Contents of this repository
+a) Alfresco Content Services deployed via [docker-compose](docker-compose/docker-compose.yml)
+1. Alfresco Repository
+2. Alfresco Share
+3. A Postgres DB  
+4. Alfresco Insight Engine (under Quay.io repository)
+5. Insight Zeppelin (under Quay.io repository)
 
-It offers you the posibility of setting affinity and tolerations for the search pod:
+b) Alfresco Search (`alfresco-search`) helm charts used as a requirement under [acs-deployment/helm/alfresco-content-services](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services) Chart
```
[the remark this answers is not in the document]
Comment (tunaaksoy) [reply README.md]: Maybe we should make it clearer that this chart on its own will not be very useful. You mentioned that it's a requirement but I'm not so sure if that's easy to understand really.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,24 @@
 # Alfresco Search Deployment
 
-This Helm chart will deploy Search Services.
+## Contents of this repository
+a) Alfresco Content Services deployed via [docker-compose](docker-compose/docker-compose.yml)
+1. Alfresco Repository
+2. Alfresco Share
+3. A Postgres DB  
+4. Alfresco Insight Engine (under Quay.io repository)
+5. Insight Zeppelin (under Quay.io repository)
```
[the remark this answers is not in the document]
Comment (paulbrodner) [reply README.md]: done!

**Comment by paulbrodner** [comment]
@tunaaksoy I've made the changes based on the review. Didn't too many settings on the Configuration table.
I've removed the affinity/tolerations settings from readme (those settings I see that are added in the [values.yaml|https://github.com/Alfresco/alfresco-search-deployment/blob/master/helm/alfresco-search/values.yaml#L63] 

If you are good with this, I can merge the changes to master.

File: README.md [hunk 3]
```diff
@@ -1,54 +1,47 @@
...
+### Configurations
+
+The following table lists the configurable parameters of the [Alfresco Search](./helm/alfresco-search) chart and their default values. 
+
+>For Alfresco Content Service chart please consult [this table](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/README.md#configuration).
+
+Parameter | Description | Default
+--- | --- | ---
+`alfresco-search.type` | Define the type of Alfresco Search to use. Available options: `insight-engine` or `search-services` | `search-services`
+`alfresco-search.registryPullSecrets` | As the Docker Image for Insight Engine is not publicly available the registryPullSecrets has to be set. More details on [SECRETS.md](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md) | NONE
```
Comment (tunaaksoy) [inline README.md]: Suggestion: Maybe we should extend the sentence and say by default this property does not have to be set as Search Services do not need the registryPullSecrets.

File: README.md [hunk 4]
```diff
@@ -1,54 +1,47 @@
...
+The following table lists the configurable parameters of the [Alfresco Search](./helm/alfresco-search) chart and their default values. 
+
+>For Alfresco Content Service chart please consult [this table](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/README.md#configuration).
+
+Parameter | Description | Default
+--- | --- | ---
+`alfresco-search.type` | Define the type of Alfresco Search to use. Available options: `insight-engine` or `search-services` | `search-services`
+`alfresco-search.registryPullSecrets` | As the Docker Image for Insight Engine is not publicly available the registryPullSecrets has to be set. More details on [SECRETS.md](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md) | NONE  
+`alfresco-search.ingress.enabled` | Enable external access for Alfresco Search Services | `true`
+`alfresco-search.ingress.basicAuth` | if `ingress.enabled=true`, user needs to provide a base64 encoded htpasswd format user name & password (ex: echo -n "$(htpasswd -nbm solradmin somepassword)" where solradmin is username and somepassword is the password) | NONE
```
Comment (tunaaksoy) [inline README.md]: Shall we copy the description from here? https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/README.md#configuration

File: README.md [hunk 5]
```diff
@@ -1,54 +1,47 @@
...
+`alfresco-search.ingress.whitelist_ips` | if `ingress.enabled=true`, user can restrict /solr to a list of IP addresses of CIDR notation | `0.0.0.0/0`
+`alfresco-search.alfresco-insight-zeppelin.enabled` | Enabled Alfresco Insight Zeppelin | `false`
```
Comment (tunaaksoy) [inline README.md]: Maybe a note saying Zeppelin will only work with Insight Engine?

File: README.md [hunk 6]
```diff
@@ -1,54 +1,47 @@
...
+* [Deploying with Helm charts on AWS using Kops](https://github.com/Alfresco/acs-deployment/tree/master/docs/helm-deployment-aws_kops.md)
+* [Deploying with Helm charts on AWS using EKS](https://github.com/Alfresco/acs-deployment/tree/master/docs/helm-deployment-aws_eks.md)
+
+### Configurations
+
+The following table lists the configurable parameters of the [Alfresco Search](./helm/alfresco-search) chart and their default values. 
+
+>For Alfresco Content Service chart please consult [this table](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/README.md#configuration).
+
+Parameter | Description | Default
```
[the remark this answers is not in the document]
Comment (paulbrodner) [reply README.md]: yes!

File: README.md [hunk 7]
```diff
@@ -1,54 +1,47 @@
...
+a) Alfresco Content Services deployed via [docker-compose](docker-compose/docker-compose.yml)
+1. Alfresco Repository
+2. Alfresco Share
+3. A Postgres DB  
+4. Alfresco Insight Engine (Quay.io access is required. Please [contact support](http://support.alfresco.com/) if you don’t yet have it)
+5. Alfresco Insight Zeppelin (Quay.io access is required. Please [contact support](http://support.alfresco.com/) if you don’t yet have it)
+
+b) Alfresco Search ([alfresco-search](./helm/alfresco-search)) helm charts used as a requirement under [acs-deployment/helm/alfresco-content-services](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services) Chart
+
+>:exclamation: you cannot deploy Alfresco Search helm chart by its own, you will need the [acs-deployment/helm/alfresco-content-services](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services) helm chart for this.
```
[the remark this answers is not in the document]
Comment (paulbrodner) [reply README.md]: done in https://github.com/Alfresco/alfresco-search-deployment/pull/7/commits/c12d8a0863f9a1934674231b432325da989990c6

File: README.md (hunk 3 above)
Comment (paulbrodner) [reply README.md]: done in https://github.com/Alfresco/alfresco-search-deployment/pull/7/commits/c12d8a0863f9a1934674231b432325da989990c6

File: README.md [hunk 8]
```diff
@@ -1,54 +1,47 @@
...
+
+## Contributing guide
+Please use [this guide](CONTRIBUTING.md) to make a contribution to the project and information to report any issues.
+
+This project contains the code for starting the entire Alfresco Content Services (Enterprise) product with **Docker** or **Kubernetes**.
+
+## Other Information
+* Checkout [acs-deployment](https://github.com/Alfresco/acs-deployment/blob/master/README.md#other-information) readme.
+
+## Blog Posts
```
[the remark this answers is not in the document]
Comment (paulbrodner) [reply README.md]: done in https://github.com/Alfresco/alfresco-search-deployment/pull/7/commits/c12d8a0863f9a1934674231b432325da989990c6

**Comment by paulbrodner** [comment]
@tunaaksoy I've updated the MR based on your suggestions. If you are happy with it, I can merge it to master! 

Thank you,

File: README.md [hunk 9]
```diff
@@ -1,54 +1,40 @@
...
+### Configurations
+
+The following table lists the configurable parameters of the [Alfresco Search](./helm/alfresco-search) chart and their default values. 
+
+>For Alfresco Content Service chart please consult [this table](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/README.md#configuration).
+
+Parameter | Description | Default
+--- | --- | ---
+`alfresco-search.type` | Define the type of Alfresco Search to use. Available options: `insight-engine` or `search-services` | `search-services`
+`alfresco-search.registryPullSecrets` | As the Docker Image for Insight Engine is not publicly available the registryPullSecrets has to be set. More details on [SECRETS.md](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md). (_By default, you don't need to set this property if your are using only the SearchServices image from [DockerHub](https://hub.docker.com/r/alfresco/alfresco-search-services/tags/) - this is public available._) | NONE
```
Comment (tunaaksoy) [inline README.md]: Suggestion: This property does not have to be set if `alfresco-search.type` is not set to `insight-engine` explicitly as the Docker image in [DockerHub](https://hub.docker.com/r/alfresco/alfresco-search-services/tags/) is publicly available.

**Comment by tunaaksoy** [comment]
I made one last suggestion. The PR can be merged after considering the last review comment. Thanks.

File: README.md (hunk 9 above)
Comment (paulbrodner) [reply README.md]: done in  e33a137

**Pull request merged**